service/src/main/resources/dev-data/employees.sql

```diff
@@ -14,6 +14,8 @@ INSERT INTO employee (id, first_name, last_name, email, external_id) VALUES
     ('00000000-0000-0000-0005-000000000001', 'Kalle', 'Kasvattaja', 'kalle.kasvattaja@espoo.fi', 'espoo-ad:00000000-0000-0000-0005-000000000001'),
     ('00000000-0000-0000-0006-000000000000', 'Erkki', 'Erityisopettaja', 'erkki.erityisopettaja@espoo.fi', 'espoo-ad:00000000-0000-0000-0006-000000000000');
 
+INSERT INTO message_account (employee_id) SELECT id FROM employee;
```

Comment:
Only supervisors and special education teachers should have an employee message account.

Reply:
That's not entirely true because global roles (at least admins and service workers) will have messaging features too. But since we will have some logic restricting accounts to certain roles, it's a good idea to create some employees without accounts to mimic that functionality.